src/aelfrice/hook.py

```
@@ -2702,15 +2710,19 @@ def session_start(
-        # Drain stdin so the hook protocol is honored. We do read the
-        # session_id from the payload (best-effort) for audit-log
-        # cross-reference; no other fields are consumed.
+        # Drain stdin so the hook protocol is honored. We read the
+        # session_id (audit cross-reference) and, on a post-compaction
+        # fire (#1031), the `source`/`cwd`/`transcript_path` fields the
+        # rebuild path needs.
         raw = ""
         try:
             raw = sin.read()
         except Exception:
             pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)